src/components/App.styl

```diff
@@ -0,0 +1,2 @@
+.App__content
+    flex-shrink 0 !important
```

Comment:
What is the problem otherwise here ?

Reply:
The header was not growing with its content. But it was a terrible solution, I found a better one by using `flex-basis` and `flex-shrink` on the header.